plugins/techdocs-node/api-report.md

```diff
@@ -58,7 +57,7 @@ export type GeneratorRunOptions = {
   outputDir: string;
   parsedLocationAnnotation?: ParsedLocationAnnotation;
   etag?: string;
-  logger: Logger;
+  logger: LoggerService;
```

Comment:
This is a bit tricky, it's a breaking change because it's used as an output type here and `Logger` is a superset of `LoggerService`. Not entirely sure what the best way forward is here tbh as it's also used in the extension point for the new backend system. We may need to simply add a separate somewhat silly separate properly. Another option might be to create a new extension point instead and deprecate the existing one.

Reply:
We decided to keep this as `Logger` from `winston`, because this is not a traditional logger service. This is actually always a Winston logger, created specifically for the generator to be able to send the logs back to the client, see https://github.com/backstage/backstage/pull/6341

Reply:
Changes reverted 🙏🏻